### DISCUSSION TITLE
Pour l'ajout d'image dans dossier

### DISCUSSION THREAD

**Discussion by CollRen** [582-41E-MA/projet-web-2-CollRen#76] [Q&A]
@claraquintela 

Bonjour Clara, est-ce qu'il y a un dossier à ajouter, ou un code à faire pour permettre l'ajout d'image. Il me semble qu'on faisait un code dans la console pour créer un lien entre React et notre dossier avec Maxime tu te souviens ?

**Comment by claraquintela** [comment]
L’unique chose que j’ai ajouté c’était la bibliothèque multer

Rien de code nécessaire:)

**Comment by claraquintela** [comment]
@CollRen , Je change aussi le create voiture. 
Je te dirais quand je termine :)

**Reply by CollRen** [reply]
Ok, pense à accepter mon PR, il y a l'ajustement des fetch dedans...
